orgs/beta.json

```diff
@@ -0,0 +1,21 @@
+{
+  "orgName": "Declarative Lookup Rollup Summaries Tool - Beta Test Org",
```

Comment:
Using this config and the one below would result in a German org for me, depending which dev hub I use. 
Should we add 
`country` and `language` properties to help international Devs to end up with an English org setup - no matter which dev hub is used?

Reply:
We certainly can. Is DLRS language-sensitive? I'm not sure what the impact would be of having development done in a non-English org; would that muck up the CustomObjectTranslations for any new schema?

Reply:
Yes, as far as I can remember this is exactly what happens.